_data-prepper/simple-pipelines.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+title: Simple pipelines
+nav_order: 12
+---
+
+This tutorial walks you through creating a simple Data Prepper pipeline and how to configure it.
+
+# Installation
+
+First, install via Docker using the following command:
+
+```
+docker pull opensearchproject/data-prepper:latest
+```
+
+# Simple configuration
+
+To help you get started, we have a simple configuration which you can use to run Data Prepper. It generates random strings and writes them to stdout. 
+
+Create a Data Prepper configuration file, `data-prepper-config.yaml`, with:
+
+```
+ssl: false
```

Comment:
I don't think we want to have documentation with `ssl: false`. We have two options here:

1. Update this to work with SSL true. It should be possible with the default values because Data Prepper includes a demo SSL certificate. So the requests below can probably be made to work against `https://`
2. Add the boilerplate about how this is insecure and should not be used in production. I know I've seen this somewhere on opensearch.org.

Reply:
Changed to "Note that this is insecure and should not be used in production." until specific boilerplate phrasing can be found. @cmanning09